routes/routes.js

```diff
@@ -0,0 +1,43 @@
+const express = require('express');
+require("dotenv").config();
+const fs = require('fs');
+const Router = module.exports = express.Router();
+
+var multer  = require('multer');
+const uploadDirectory = `${__dirname}/../uploads/`;
+
+var upload = multer({ dest: uploadDirectory });
+
+const AWS = require('aws-sdk');
+const s3 = new AWS.S3();
+
+Router.post('/resource', upload.single(`lab18file`) ,(req, res, next) => {
+    const base64data = new Buffer(req.file.destination, 'binary');
+    fs.readFile(`${__dirname}/../uploads/${req.file.filename}`, (err, data) => {
```

Comment:
I would prefer seeing this section factored out into a separate module so that the route itself is only concerned with physical input/output, not the business logic of handling the actual files.